dts/board format property node order

kylebonnici opened the pull request:
This PR addresse the style guide issues to ensure properties are on top on nodes and that properties are sorted as per

(https://docs.kernel.org/devicetree/bindings/dts-coding-style.html)
```
Order of Properties in Device Node
The following order of properties in device nodes is preferred:

“compatible”
“reg”
“ranges”
Standard/common properties (defined by common bindings, e.g. without vendor-prefixes)
Vendor-specific properties
“status” (if applicable)
Child nodes, where each node is preceded with a blank line
```

This PR does not address sorting of nodes.

The style guide says

```
Nodes on any bus, thus using unit addresses for children, shall be ordered by unit address in ascending order. Alternatively for some subarchitectures, nodes of the same type can be grouped together, e.g. all I2C controllers one after another even if this breaks unit address ordering.
```


If sorting of nodes is also desired, to avoid parsing bindings for linting to determine the bus type, "Nodes on any bus, thus using unit addresses for children, shall be ordered by unit address in ascending order." would be the only feasible solution IMO

UPDATE

First 4 commits shell be ignored as these are part of another PR https://github.com/zephyrproject-rtos/zephyr/pull/101619

yishai1999 commented:
Is this enforced in the current version of linter?

kylebonnici commented on boards/silabs/dev_kits/xg24_dk2601b/xg24_dk2601b.dts:
```diff
@@ -189,19 +189,19 @@
-	status = "okay";
 
 	cs-gpios = <&gpioc 0 GPIO_ACTIVE_LOW>;
+	status = "okay";
```
Currently we have "4 blocks" 

Block 1:
- “compatible”
- “reg”
- “ranges”

Block 2:
- Standard/common properties (defined by common bindings, e.g. without vendor-prefixes)

Block 3:
- Vendor-specific properties

Block 4:
- “status” (if applicable)

if we are to do something about `status ` perhaps should do some thing about the other blocks.

I am open to this if this if the coding standard we want to apply.

@nordicjm, @fabiobaltieri, @kartben , @keith-zephyr what are your thoughts

nordicjm commented:
> @nordicjm so in your opinion should this be something we enforce in CI as well?

Linux specifies it is preferred https://docs.kernel.org/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node so I assume it is not required, but do not know if that is enforced by maintainers

kylebonnici committed:
docs: Update Devicetree style guide

Order of Properties in Device Node

The following order of properties in device nodes is required:

“compatible”
“reg”
“ranges”
Standard/common properties (defined by common bindings,
                            e.g. without vendor-prefixes)
Vendor-specific properties
“status” (if applicable)
Child nodes, where each node is preceded with a blank line

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici commented:
@kartben  docs updated. If this will become mandatory do we want to update all files that are not in the boards folders in this PR or we should split like we had done for the other formatting PR?

kylebonnici committed:
style: format files in boards/01space

sorting properties and nodes

boards/01space

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/actinius

sorting properties and nodes

boards/actinius

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/aconno

sorting properties and nodes

boards/aconno

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/96boards

sorting properties and nodes

boards/96boards

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/acrn

sorting properties and nodes

boards/acrn

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/adafruit

sorting properties and nodes

boards/adafruit

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/aesc

sorting properties and nodes

boards/aesc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/adi

sorting properties and nodes

boards/adi

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/alientek

sorting properties and nodes

boards/alientek

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/aithinker

sorting properties and nodes

boards/aithinker

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/alif

sorting properties and nodes

boards/alif

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/amd

sorting properties and nodes

boards/amd

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ambiq

sorting properties and nodes

boards/ambiq

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/andestech

sorting properties and nodes

boards/andestech

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/antmicro

sorting properties and nodes

boards/antmicro

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/arduino

sorting properties and nodes

boards/arduino

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/aspeed

sorting properties and nodes

boards/aspeed

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/atmarktechno

sorting properties and nodes

boards/atmarktechno

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/arm

sorting properties and nodes

boards/arm

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/bbc

sorting properties and nodes

boards/bbc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/bcdevices

sorting properties and nodes

boards/bcdevices

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/atmel

sorting properties and nodes

boards/atmel

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/beagle

sorting properties and nodes

boards/beagle

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/bflb

sorting properties and nodes

boards/bflb

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/blues

sorting properties and nodes

boards/blues

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/blackberry

sorting properties and nodes

boards/blackberry

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/brcm

sorting properties and nodes

boards/brcm

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/contextualelectronics

sorting properties and nodes

boards/contextualelectronics

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/cdns

sorting properties and nodes

boards/cdns

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/bytesatwork

sorting properties and nodes

boards/bytesatwork

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/cirrus

sorting properties and nodes

boards/cirrus

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/circuitdojo

sorting properties and nodes

boards/circuitdojo

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/coredevices

sorting properties and nodes

boards/coredevices

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ct

sorting properties and nodes

boards/ct

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/cypress

sorting properties and nodes

boards/cypress

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/croxel

sorting properties and nodes

boards/croxel

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/dfrobot

sorting properties and nodes

boards/dfrobot

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/cytron

sorting properties and nodes

boards/cytron

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/digilent

sorting properties and nodes

boards/digilent

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/doiting

sorting properties and nodes

boards/doiting

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/dragino

sorting properties and nodes

boards/dragino

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ebyte

sorting properties and nodes

boards/ebyte

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/dptechnics

sorting properties and nodes

boards/dptechnics

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/efinix

sorting properties and nodes

boards/efinix

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/elan

sorting properties and nodes

boards/elan

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/enclustra

sorting properties and nodes

boards/enclustra

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/egis

sorting properties and nodes

boards/egis

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/electronut

sorting properties and nodes

boards/electronut

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/element14

sorting properties and nodes

boards/element14

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/embedsky

sorting properties and nodes

boards/embedsky

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ene

sorting properties and nodes

boards/ene

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/enjoydigital

sorting properties and nodes

boards/enjoydigital

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/espressif

sorting properties and nodes

boards/espressif

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/firefly

sorting properties and nodes

boards/firefly

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/fanke

sorting properties and nodes

boards/fanke

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ezurio

sorting properties and nodes

boards/ezurio

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/franzininho

sorting properties and nodes

boards/franzininho

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/fysetc

sorting properties and nodes

boards/fysetc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/fobe

sorting properties and nodes

boards/fobe

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/focaltech

sorting properties and nodes

boards/focaltech

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/framework

sorting properties and nodes

boards/framework

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/gardena

sorting properties and nodes

boards/gardena

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/gaisler

sorting properties and nodes

boards/gaisler

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/gd

sorting properties and nodes

boards/gd

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/google

sorting properties and nodes

boards/google

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/heltec

sorting properties and nodes

boards/heltec

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/hardkernel

sorting properties and nodes

boards/hardkernel

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/iar

sorting properties and nodes

boards/iar

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/holyiot

sorting properties and nodes

boards/holyiot

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/infineon

sorting properties and nodes

boards/infineon

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/innblue

sorting properties and nodes

boards/innblue

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/intel

sorting properties and nodes

boards/intel

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ite

sorting properties and nodes

boards/ite

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/khadas

sorting properties and nodes

boards/khadas

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/kincony

sorting properties and nodes

boards/kincony

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/kws

sorting properties and nodes

boards/kws

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/longan

sorting properties and nodes

boards/longan

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/lowrisc

sorting properties and nodes

boards/lowrisc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/lilygo

sorting properties and nodes

boards/lilygo

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/luatos

sorting properties and nodes

boards/luatos

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/makerbase

sorting properties and nodes

boards/makerbase

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/makerdiary

sorting properties and nodes

boards/makerdiary

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/m5stack

sorting properties and nodes

boards/m5stack

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/madmachine

sorting properties and nodes

boards/madmachine

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/mediatek

sorting properties and nodes

boards/mediatek

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/mikroe

sorting properties and nodes

boards/mikroe

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/microchip

sorting properties and nodes

boards/microchip

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/mxchip

sorting properties and nodes

boards/mxchip

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/native

sorting properties and nodes

boards/native

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/norik

sorting properties and nodes

boards/norik

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/nordic

sorting properties and nodes

boards/nordic

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/nuvoton

sorting properties and nodes

boards/nuvoton

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/nxp

sorting properties and nodes

boards/nxp

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/oct

sorting properties and nodes

boards/oct

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/olimex

sorting properties and nodes

boards/olimex

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/openisa

sorting properties and nodes

boards/openisa

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/openhwgroup

sorting properties and nodes

boards/openhwgroup

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/others

sorting properties and nodes

boards/others

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/panasonic

sorting properties and nodes

boards/panasonic

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/particle

sorting properties and nodes

boards/particle

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/peregrine

sorting properties and nodes

boards/peregrine

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/pcbcupid

sorting properties and nodes

boards/pcbcupid

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/phytec

sorting properties and nodes

boards/phytec

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/pimoroni

sorting properties and nodes

boards/pimoroni

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/pjrc

sorting properties and nodes

boards/pjrc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/pine64

sorting properties and nodes

boards/pine64

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/qorvo

sorting properties and nodes

boards/qorvo

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/qualcomm

sorting properties and nodes

boards/qualcomm

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/qemu

sorting properties and nodes

boards/qemu

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/quicklogic

sorting properties and nodes

boards/quicklogic

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/raspberrypi

sorting properties and nodes

boards/raspberrypi

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/rakwireless

sorting properties and nodes

boards/rakwireless

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/raytac

sorting properties and nodes

boards/raytac

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/realtek

sorting properties and nodes

boards/realtek

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/renesas

sorting properties and nodes

boards/renesas

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/renode

sorting properties and nodes

boards/renode

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/retronix

sorting properties and nodes

boards/retronix

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ruuvi

sorting properties and nodes

boards/ruuvi

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ronoth

sorting properties and nodes

boards/ronoth

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ruiside

sorting properties and nodes

boards/ruiside

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sc

sorting properties and nodes

boards/sc

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/seco

sorting properties and nodes

boards/seco

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/seagate

sorting properties and nodes

boards/seagate

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/segger

sorting properties and nodes

boards/segger

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sensry

sorting properties and nodes

boards/sensry

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/seeed

sorting properties and nodes

boards/seeed

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/shields

sorting properties and nodes

boards/shields

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sifive

sorting properties and nodes

boards/sifive

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sifli

sorting properties and nodes

boards/sifli

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/silabs

sorting properties and nodes

boards/silabs

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sipeed

sorting properties and nodes

boards/sipeed

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/solderedelectronics

sorting properties and nodes

boards/solderedelectronics

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/snps

sorting properties and nodes

boards/snps

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/sparkfun

sorting properties and nodes

boards/sparkfun

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/starfive

sorting properties and nodes

boards/starfive

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/st

sorting properties and nodes

boards/st

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/tdk

sorting properties and nodes

boards/tdk

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/technexion

sorting properties and nodes

boards/technexion

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/steelseries

sorting properties and nodes

boards/steelseries

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/thirdreality

sorting properties and nodes

boards/thirdreality

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/telink

sorting properties and nodes

boards/telink

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/ti

sorting properties and nodes

boards/ti

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/toradex

sorting properties and nodes

boards/toradex

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/u-blox

sorting properties and nodes

boards/u-blox

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/udoo

sorting properties and nodes

boards/udoo

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/up-bridge-the-gap

sorting properties and nodes

boards/up-bridge-the-gap

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/variscite

sorting properties and nodes

boards/variscite

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/vicharak

sorting properties and nodes

boards/vicharak

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/vcc-gnd

sorting properties and nodes

boards/vcc-gnd

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/vngiotlab

sorting properties and nodes

boards/vngiotlab

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/waveshare

sorting properties and nodes

boards/waveshare

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/viewe

sorting properties and nodes

boards/viewe

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/wch

sorting properties and nodes

boards/wch

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/we

sorting properties and nodes

boards/we

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/weact

sorting properties and nodes

boards/weact

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/wemos

sorting properties and nodes

boards/wemos

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/witte

sorting properties and nodes

boards/witte

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/xen

sorting properties and nodes

boards/xen

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/xunlong

sorting properties and nodes

boards/xunlong

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kylebonnici committed:
style: format files in boards/wiznet

sorting properties and nodes

boards/wiznet

Signed-off-by: Kyle Bonnici <kylebonnici@hotmail.com>

kartben commented:
Can't make a proper "suggested change" due to how large the PR is an GH choking on it but you have fancy double quotes and incorrect rst formatting (see https://builds.zephyrproject.io/zephyr/pr/96317/docs/contribute/style/devicetree.html)

```rst
  * Device node properties are **required** to appear in the following order:

    * "compatible"
    * "reg"
    * "ranges"
    * Standard/common properties (defined by common bindings, e.g. without vendor prefixes)
    * Vendor-specific properties
    * "status" (if applicable)
    * Child nodes
```

I would also recommend updating the provided example so that it's compliant ;-) (and maybe make it slightly more "rich" so that it also features things like reg, ranges, etc.
